pic-domain/src/main/kotlin/com/mashup/pic/domain/user/User.kt

```diff
@@ -0,0 +1,30 @@
+package com.mashup.pic.domain.user
+
+import com.mashup.pic.domain.common.BaseEntity
+import jakarta.persistence.*
+import org.hibernate.annotations.SQLDelete
+import org.hibernate.annotations.SQLRestriction
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "users")
+@SQLDelete(sql = "UPDATE users SET deleted_at = CURRENT_TIMESTAMP WHERE id = ?")
+@SQLRestriction("deleted_at is NULL")
+class User(
+
+        @Column(name = "oauth_id", nullable = false)
+        val oAuthId: Long = 0,
+
+        @Column(name = "nickname", nullable = false)
+        val nickname: String = "",
+
+        @Column(name = "profileImage", nullable = false)
+        val profileImage: String = "",
```

Comment:
```suggestion
        @Column(name = "nickname", nullable = false)
        val nickname: String,

        @Column(name = "profileImage", nullable = false)
        val profileImage: String,
```
nullable false이고 default value 없으면 필드도 필수로 적을 수 있게 하시죠!

Reply:
생성자를 따로 어떻게 만들자는 말씀이시죵 ?